addons/hr_holidays/static/src/tours/hr_holidays_tour.js

```diff
@@ -82,6 +82,12 @@ registry.category("web_tour.tours").add('hr_holidays_tour', {
     {
         trigger: 'button[name="action_approve"]',
         content: _t("Let's approve it"),
-        position: 'bottom'
+        position: 'bottom',
+    },
+    {
+        trigger: `tr.o_data_row:first:not(:has(button[name="action_approve"]))`,
+        content: _t("Varify leave is approved"),
```

Comment:
:detective: Typo
And the translation is actually useless on an `auto: true` step since it's only run in tests, not shown to users.
Arguably you could even omit the `content` property but I prefer to keep it, it's easier when debugging tours.